websites/docs-smoke-test/src/content/components/cards.mdx

```diff
@@ -0,0 +1,203 @@
+---
+title: Cards
+---
+
+ * The actual card content is markdown
+ * Title have no linkable / hover anchor like section headings
+ * Titles never become entries in the page index nav like section headings
```

Comment:
That boils down to an accessibility question - they would not appear in our index nav in any case because that carries only native markdown headings anyways. 

Here's a full in-depth post on accesibilty vs. cards:  https://inclusive-components.design/cards/   .  Esp. the clickability thing seems to be ugly but on the other hand optional, too.   

If we go for making the cards headings I suggest H5 and H6 so they are always below any heading that we allow to author in markdown.

Reply:
My main "concern" is if we use headings, would the generated HTML also change, given all our plugins etc that "modify" and wrap things around the headings?

Do you know what I mean?

Reply:
No, the section wrapping would not catch these headings because it works on the MDX AST which only sees the Card component JSX as it's in the MDX.   The AST transforms never see the output of the components - that's why for example in the API types we chose to not generate a heading from the dynamic component, the authors have to put an actual MD heading above the component.